tests/VirtoCommerce.CatalogPersonalizationModule.Test/VirtoCommerce.CatalogPersonalizationModule.Test.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+      <noWarn>1591</noWarn>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
+    <PackageReference Include="MockQueryable.Moq" Version="3.0.2" />
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="VirtoCommerce.ExportModule.CsvProvider" Version="3.0.0-rc.2" />
```

Comment:
Why do we need export module for the tests?

Reply:
removed